larq_compute_engine/core/bconv2d_output_transform.h

```diff
@@ -6,7 +6,7 @@
 #include <limits>
 
 #include "tensorflow/lite/kernels/internal/common.h"
-#include "tensorflow/lite/kernels/internal/round.h"
+#include "tensorflow/lite/kernels/internal/reference/round.h"
```

Comment:
is this a reference impl. for rounding? are there other impl.s for rounding as well?

Reply:
Good catch, I think this include might be wrong or not necessary, I'll ceck.

Reply:
They moved the definition in https://github.com/tensorflow/tensorflow/commit/a4c2fcf05b831c726346d78b4b3dddfd823d93c0 so `reference/round.h` but I guess the correct header was implicitely included in `common.h`.